components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -54,7 +54,12 @@ class Session(
         fun onProgress(session: Session, progress: Int) = Unit
         fun onLoadingStateChanged(session: Session, loading: Boolean) = Unit
         fun onNavigationStateChanged(session: Session, canGoBack: Boolean, canGoForward: Boolean) = Unit
-        fun onLoadRequest(session: Session, triggeredByRedirect: Boolean, triggeredByWebContent: Boolean) = Unit
+        fun onLoadRequest(
+            session: Session,
+            url: String,
```

Comment:
nit: I think this is a breaking change, should it be added to the log change file?